src/client/components/dragAndDrop/drag-and-drop.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Pizzicato from 'pizzicato';
+import Dropzone from 'react-dropzone';
+import { createAudioData, playPauseSoundFromFile } from '../../actions/audioActions';
+
+import './drag-and-drop.css';
+
+
+class DragAndDrop extends Component {
+    onDrop = (acceptedFiles) => {
+      const file = acceptedFiles[0];
+      const audioFile = new Audio(URL.createObjectURL(file));
+      const sound = new Pizzicato.Sound({
+        source: 'file',
+        options: {
+          path: audioFile.src,
+          loop: true,
+        },
+      }, () => this.createSoundInfoInState(sound, file));
+    };
+
+  createSoundInfoInState = (sound, file) => {
+    const { audioData: { analyser }, createAudioDataAsProp } = this.props;
+    sound.connect(analyser);
+    createAudioDataAsProp({
+      sound,
+      trackName: file.name,
+      trackType: file.type,
```

Comment:
We already don't use trackName and trackType, something goes wrong with your merge :(

Reply:
done